src/ray/rpc/raylet/raylet_client.h

```diff
@@ -35,15 +36,20 @@ namespace rpc {
 using ResourceMappingType =
     std::unordered_map<std::string, std::vector<std::pair<int64_t, double>>>;
 
-/// Client used for communicating with raylet.
+/// Client used for communicating with the raylet.
 class RayletClient {
  public:
-  /// Constructor for raylet client.
+  /// Constructor for the raylet client.
   /// TODO(jzh): At present, client call manager and reply handler service are generated
-  /// in raylet client. Should add parameters to the constructor and pass them in.
-  /// Change them as input parameters once we changed the worker into a server.
+  /// in raylet client. Instead, we should add parameters to the constructor and pass them
+  /// in. Change them as input parameters once we changed the worker into a server.
   ///
   /// \param[in] raylet_socket Unix domain socket of the raylet server.
+  /// \param[in] worker_id The worker id.
+  /// \param[in] is_worker Indicates whether a worker or a driver.
+  /// \param[in] job_id The job id that this raylet client belongs to.
+  /// \param[in] language The language type, python or java.
+  /// \param[in] port The listening port of the worker server.
```

Comment:
I know it wasn't added by you, but could you add a brief comment here explaining what the special values are (e.g., -1 means that the worker does not have a server).